contracts/upgradeable_contracts/RewardableBridge.sol

```diff
@@ -0,0 +1,63 @@
+pragma solidity 0.4.24;
+
+import "./OwnedUpgradeability.sol";
+import "./Ownable.sol";
+
+
+contract RewardableBridge is Ownable, OwnedUpgradeability {
+
+    function setFee(uint256 _fee) external onlyIfOwnerOfProxy {
+        require(feeManagerContract().delegatecall(abi.encodeWithSignature("setFee(uint256)", _fee)));
+    }
+
+    function getFee() public view returns(uint256) {
+        uint256 fee;
+        bytes memory callData = abi.encodeWithSignature("getFee()");
+        address feeManager = feeManagerContract();
+        assembly {
+            let result := delegatecall(gas, feeManager, add(callData, 0x20), mload(callData), 0, 32)
```

Comment:
will it be safer to call `callcode` here?

Reply:
Updated to use `callcode` https://github.com/poanetwork/poa-bridge-contracts/pull/132/commits/6b9ce95bccb7dbfa584935c9d198248d2a7b00a0